tfjs-backend-webgpu/src/flags_webgpu.ts

```diff
@@ -31,7 +31,7 @@ ENV.registerFlag('WEBGPU_CPU_FORWARD', () => true);
 /**
  * Thread register block size for matmul kernel.
  */
-ENV.registerFlag('WEBGPU_MATMUL_WORK_PER_THREAD', () => 4);
+ENV.registerFlag('WEBGPU_MATMUL_WORK_PER_THREAD', () => 2);
```

Comment:
I changed the workgroup size from (8, 8, 1) to (16, 16, 1) to better align with bank size 16. However, due to the maximum shared memory size limitation, I have to change the work per thread size to 2.

Reply:
Remove this part of changes to make sure this PR only focuses on fixing bank conflicts. Please take another look, thanks.